src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx

```diff
@@ -127,6 +128,17 @@ export const ProjectDetailsPage: React.FC = () => {
           </CreateCollapse.Panel>
           <CreateCollapse.Panel
             key={2}
+            header={<OptionalHeader header={t`Project Tags`} />}
+            hideDivider
+          >
+            <FormItems.ProjectTags
+              name="tags"
+              hideLabel
+              initialTags={formProps.form.getFieldValue('tags')}
```

Comment:
@wraeth-eth is there a better way we can handle this?

Reply:
probably... but I couldn't figure out anything better that actually worked with FormItems. this pattern is used in several places in the app so i just went with it

Reply:
Hey @peripheralist , I had a stab at this here:
https://github.com/jbx-protocol/juice-interface/commit/9c8af8d0d18dccccda25803865071f45e5aad64b

Take a look and lemme know what you think :)